app/viewmodels/checkAnswers/sections/items/ItemPackagingSummary.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.checkAnswers.sections.items
+
+import models.Index
+import models.requests.DataRequest
+import models.response.referenceData.ItemPackaging
+import pages.sections.items._
+import play.api.i18n.Messages
+import play.twirl.api.{Html, HtmlFormat}
+import queries.ItemsPackagingCount
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import utils.Logging
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+import javax.inject.Inject
+
+class ItemPackagingSummary @Inject()(tag: views.html.components.tag) extends Logging {
+
+  def row(itemIdx: Index)(implicit request: DataRequest[_], messages: Messages): Option[SummaryListRow] =
+    request.userAnswers.get(ItemBulkPackagingChoicePage(itemIdx)).flatMap {
+      case true =>
+        constructBulkPackagingSummary(itemIdx)
+      case _ =>
+        getPackagesForItem(itemIdx) match {
+          case Nil => None
+          case packages =>
+            Some(SummaryListRowViewModel(
+              key = "itemsAddToList.packagesCyaLabel",
+              value = ValueViewModel(HtmlContent(constructPackagingValues(itemIdx, packages).mkString("<br>")))
+            ))
+        }
+    }
+
+  private def getPackagesForItem(itemIdx: Index)(implicit request: DataRequest[_]): Seq[(ItemPackaging, Option[String])] =
+    request.userAnswers.get(ItemsPackagingCount(itemIdx)).fold(Seq.empty[(ItemPackaging, Option[String])]) { count =>
+      (0 until count).map(Index(_)).map { packageIdx =>
+        request.userAnswers.get(ItemSelectPackagingPage(itemIdx, packageIdx)) ->
+          request.userAnswers.get(ItemPackagingQuantityPage(itemIdx, packageIdx))
+      }.collect {
+        case (Some(epc), quantity) => epc -> quantity
+      }
+    }
+
+  private def constructPackagingValues(itemIdx: Index,
+                                       packages: Seq[(ItemPackaging, Option[String])])(implicit request: DataRequest[_], messages: Messages): Seq[Html] =
+    packages.zipWithIndex.map {
+      case ((itemPackaging, quantity), packageIdx) =>
+        HtmlFormat.fill(Seq(
+          Some(Html(s"${quantity.fold("")(_ + " ")}${itemPackaging.description}")),
```

Comment:
I need to check this: does this return 1 box? Or 1x box

Reply:
Looks like it returns `<quantity> pallet`, I would have thought it would return `<quantity>x pallet`
![image](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/53828896/c98a7869-cbd8-4b39-bb79-fe566dce2590)

Reply:
According to the copydeck, there should be a comma after each packaging type if there is > 1

Reply:
I'll add in the "x", I've got an outstanding question about the comma though which I guess will be resolved in an alignment story.